src/db/migrations/20181226223221_connect_itemhistory_saldohistory.js

```diff
@@ -101,3 +88,36 @@ export const down = async (knex) => {
 		throw new Error('dont drop stuff in production');
 	}
 };
+
+const isSameMultibuy = (itemEvent, nextEvent) => {
+	if (nextEvent === undefined) {
+		throw new Error(`nextEvent is undefined for itemEvent ${JSON.stringify(itemEvent)}`);
+	}
+
+	return (
+		itemEvent.time.getTime() === nextEvent.time.getTime() &&
+		itemEvent.userid === nextEvent.userid &&
+		itemEvent.sellprice === nextEvent.sellprice
+	);
+};
+
+const isMultibuySaldoEvent = (itemEvent, saldoEvent) => {
+	if (saldoEvent === undefined) {
+		throw new Error(`saldoEvent is undefined for itemEvent ${JSON.stringify(itemEvent)}`);
+	}
+
+	// Weird edge cases in production data
+	if (
```

Comment:
We should consider differentiating multibuy events by item id instead, if possible